### ISSUE TITLE
Assets folder

### ISSUE THREAD

**Issue by marioteodoru** [K240/HoudiniEngineFor3dsMax#3]
Hello. 
It's my first time here. I can't seem to have an Assets folder in the bin\Max2016 folder

**Comment by ilayshp** [comment]
Hello
I make dir h2max in any dsk and it contains dirs:
* x:\h2max:
    * assets
    * otl
    * tx

then set up them in h2max settinngs

**Comment by marioteodoru** [comment]
hello, 
please consider i am not a specialist. i don't understand what you are saying.

**Comment by ilayshp** [comment]
Yes, any name(which 3dsmax support) for dir to store assets and maxscript plugins

**Comment by K240** [comment]
Please refer to "Houdini Engine Plugin Path (assets folder)" and "PATH environment variable" on InstallGuide_en.docx.

Thank you for help, ilayshp.
